public_html/index.html

```diff
@@ -15,6 +15,14 @@
     <!--[if lt IE 9]>
       <script src="https://html5shim.googlecode.com/svn/trunk/html5.js"></script>
     <![endif]-->
+    <meta property="og:type" content="website">
+    <meta property="og:site_name" content="Mozilla - The Future of Web Games">
```

Comment:
a recent thing I learned:

if you define these OpenGraph tags, you actually don’t need to define a lot (most?) for the Twitter Card tags.

that might make your other PR a bit simpler. YMMV.

Reply:
I didn't know about this. It's very nice! But what meta tags I need to remove from the twitter PR?